config/runtime/base/python.jinja2

```diff
@@ -62,7 +63,8 @@ if __name__ == '__main__':
     try:
         src_path = _get_source(TARBALL_URL)
         res = main(sys.argv + [src_path])
-    except:
+    except Exception as e:
```

Comment:
Do we need "as e" if we are not using it?

Reply:
Not really, I'll drop this.